shared/src/main/kotlin/org/javacs/kt/util/Utils.kt

```diff
@@ -17,8 +17,14 @@ fun execAndReadStdoutAndStderr(shellCommand: String, directory: Path): Pair<Stri
     val process = Runtime.getRuntime().exec(shellCommand, null, directory.toFile())
     val stdout = process.inputStream
     val stderr = process.errorStream
-    val output = stdout.bufferedReader().use { it.readText() }
-    val errors = stderr.bufferedReader().use { it.readText() }
+    var output = ""
+    var errors = ""
+    val outputThread = Thread { stdout.bufferedReader().use { output += it.readText() } }
+    val errorsThread = Thread { stderr.bufferedReader().use { errors += it.readText() } }
+    outputThread.start()
+    errorsThread.start()
+    outputThread.join()
+    errorsThread.join()
```

Comment:
here's an attempt at making sure this still works. the problem is that the call to `.use` on the BufferedReader for stdout is blocking, and it doesn't release until both stdout and stderr are cleared, seemingly.